### PR TITLE
identify users in FullStory

### PR DESCRIPTION
**Pull request by katehedgpeth**
Asana ticket: [Connect usernames to FullStory](https://app.asana.com/0/1112935048846093/1130673090389400)

[Fullstory documentation](https://help.fullstory.com/develop-js/137379) recommends not using strings with personal information for the UID, which is why I've encrypted the username for the UID.

Confirmed that this is working:
Screen Shot 2019-07-23 at 11 47 40 AM

### CODE REVIEW

**Comment by katehedgpeth** [comment]
@arkadyan I'm not sure how to fix the test coverage -- in the test environment, `AuthManager.Plug.current_resource(conn)` returns `%{}` instead of a string. Do you know if there's some way to change that, like some kind of config or something?

**Comment by arkadyan** [comment]
@katehedgpeth I'm pretty sure those test values are defined by our [Fake](https://github.com/mbta/skate/blob/master/lib/skate/ueberauth/strategy/fake.ex) strategy module. I haven't messed with that much so this is a guess, but I wonder if putting data into the `info/1` return will make it end up in your `current_resource` call?

**Comment by katehedgpeth** [comment]
@arkadyan turns out `%{}` was what was being passed to `AuthManager.encode_and_sign`, which I totally missed.

**Comment by arkadyan** [comment]
@katehedgpeth Oh, good catch! I guess we just didn't care previously.

File: lib/skate_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -2,11 +2,29 @@ defmodule SkateWeb.PageController do
...
+
     _ = Logger.info("uid=#{uid}")
 
-    render(conn, "index.html")
+    conn
+    |> assign(:user_info, user_info(uid))
+    |> render("index.html")
+  end
+
+  defp user_info(id) when is_binary(id) do
```
Comment (arkadyan) [inline lib/skate_web/controllers/page_controller.ex]: Is the `when is_binary(id)` guard necessary?

**Comment by arkadyan** [comment]
@katehedgpeth one more thing: should those salts be in env vars as well?

**Comment by katehedgpeth** [comment]
@arkadyan The screenshot is showing the display name. apparently that's ok to keep as PII, according to their documentation. I put the salts into env vars, good catch.

**Comment by arkadyan** [comment]
@katehedgpeth sorry I keep forgetting things, but could you please add those 2 new vars to `.env.example` as well to let folks know they are required?

**Pull request merged**